### ISSUE TITLE
Rule proposal: No multiple hooks on the same level

### ISSUE THREAD

**Issue by jfmengels** [lo1tuma/eslint-plugin-mocha#82]
In the codebase of one of my projects, I often see multiple hooks of the same type being defined under the same `describe` level:

```js
describe('foo', function() {
  before(function() {
    // set up something
  });
  before(function() {
    // set up something else
  });

  // tests
});
```

This can be confusing when one hook is before the tests and the other after the tests (think `after`/`afterEach`). Anyway, this is IMO not a good pattern as you can and should do everything in one hook.

**Comment by lo1tuma** [comment]
Totally agreed. I’ve also seen this a couple of times and was always confused.

👍
